micrometer-spring-legacy/src/main/java/io/micrometer/spring/autoconfigure/export/appoptics/AppopticsProperties.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.spring.autoconfigure.export.appoptics;
+
+import io.micrometer.spring.autoconfigure.export.StepRegistryProperties;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * {@link ConfigurationProperties} for configuring AppOptics metrics export.
+ *
+ * @author Hunter Sherman
+ */
+@ConfigurationProperties(prefix = "management.metrics.export.appoptics")
+public class AppopticsProperties extends StepRegistryProperties {
```

Comment:
[dropwizard-librato](https://github.com/librato/dropwizard-librato#tagging-support) supports static sets of tags. Do we want to support that here as well? It's kind of nice to have the ability to do that.

Reply:
I didn't add these because I believe that micrometer will handle that via https://github.com/micrometer-metrics/micrometer/pull/705#discussion_r202558483 which as I understand it will add custom tags from configuration to each metric. I may be misunderstanding it though.